service/telemetry/config.go

```diff
@@ -92,6 +92,19 @@ type LogsConfig struct {
 	//
 	// By default, there is no initial field.
 	InitialFields map[string]any `mapstructure:"initial_fields"`
+
+	// TimeEncoding controls the zap logger's time encoding
+	// Permitted values are:
+	//  - "Epoch"
+	//  - "EpochMillis"
+	//  - "EpochNanos"
+	//  - "ISO8601"
+	//  - "RFC3339"
+	//  - "RFC3339Nano"
+	// Leaving the value blank results in the default time encoding, which
+	// depends on the message encoding (for JSON, it will be EpochNanos,
+	// and for console, it will be ISO8601
+	TimeEncoding string `mapstructure:"time_encoding"`
```

Comment:
We would prefer not to derive from the zap.Config. Is this there already? if yes, can you add a link to confirm the name?

Reply:
Apologies, but I'm not following this.  I wasn't able to find any mechanism by which I can set the Zap logger's time encoding through the existing telemetry config, hence the new field.  One can only indirectly influence it (ex, by choosing console as the format), but there is no way to use, say, RFC3339 time encoding along with JSON formatted messages.